aptos-move/block-executor/src/executor.rs

```diff
@@ -538,53 +545,39 @@ where
                         finalized_groups.push((group_key, metadata_op, finalized_group));
                     },
                     Err(err) => {
-                        maybe_code_error = Err(err.into());
-                        break;
+                        return Err(err.into());
                     },
                 }
             }
 
-            if maybe_code_error.is_ok() {
-                if let Some(group_reads_needing_delayed_field_exchange) =
-                    last_input_output.group_reads_needing_delayed_field_exchange(txn_idx)
+            if let Some(group_reads_needing_delayed_field_exchange) =
```

Comment:
the "needing" is really poking my eyes :) -> "require"

Reply:
I'm not changing it in this PR, that change would touch 30 files.

Reply:
if @igor-aptos though, I can send a PR to bulk replace, not lazy, not a problem :)

Reply:
I'm fine with group_reads_requiring_delayed_field_exchange, but for my English- what is wrong with "needing"? Sounds like correct English to me ? :)

Reply:
it's "needling" my eyes :) Dunno, just sounds off (and require is kind of standard)